c++/src/kj/async.c++

```diff
@@ -1460,21 +1463,25 @@ FiberStack::FiberStack(size_t stackSizeParam)
   if (_setjmp(impl->originalJmpBuf) == 0) {
     setcontext(&context);
   }
+#endif
 #else
   KJ_UNIMPLEMENTED(
       "Fibers are not implemented on this platform because its C library lacks setcontext() "
       "and friends. If you'd like to see fiber support added, file a bug to let us know. "
       "We can likely make it happen using assembly, but didn't want to try unless it was "
       "actually needed.");
 #endif
+#endif
```

Comment:
I can't figure out what `#if` this is matching. Maybe time to add comments to the `#endif`s to make clear? Like:

```
#endif  // KJ_USE_FIBERS
```

Reply:
Oh I see it's for KJ_NO_EXCEPTIONS. Perhaps we should roll that into `KJ_USE_FIBERS`? I.e. make sure to define `KJ_USE_FIBERS 0` when `KJ_NO_EXCEPTIONS` is enabled.

Reply:
Done. It's bit hairy now that you start considering all permutations of what might be defined and what not.... But let's see if you like it and go from there. )